### ISSUE TITLE
Auth0: Issued At (iat) claim error in the ID token

### ISSUE THREAD

**Issue by lappalj4** [hackjunction/hackplatform#29]
Some users are (occasionally) seeing the following error when trying to log in: 

```
Issued At (iat) claim error in the ID token; current time "Wed Jan 15 2020 22:26:59 GMT+0100 (heure normale d’Europe centrale)" is before issued at time "Wed Jan 15 2020 22:27:01 GMT+0100 (heure normale d’Europe centrale)"
```

where the Issued At field in the ID token is off by a couple of seconds. We should try and find a way to allow a small difference in the issued at field of the token. 

This error is thrown by `handleAuthentication` in `frontend/src/redux/auth/actions`, so perhaps there is a possibility to configure the Auth0 library with a small "leeway". Discussion on the matter here: https://community.auth0.com/t/issued-at-claim-error/34291/9

**Comment by lappalj4** [comment]
Solution here (upgrade Auth0-js library in the frontend): https://github.com/auth0/auth0.js/issues/1061#issuecomment-566771343

**Comment by lappalj4** [comment]
Fixed in #30
